src/Stripe.net/Services/Files/FileService.cs

```diff
@@ -26,11 +23,15 @@ public FileService(string apiKey)
 
         public virtual File Create(FileCreateOptions options, RequestOptions requestOptions = null)
         {
+            requestOptions = this.SetupRequestOptions(requestOptions);
+            requestOptions.BaseUrl = requestOptions.BaseUrl ?? StripeConfiguration.FilesBase;
```

Comment:
I don't fully grasp that line and why it's needed. Since we always know we want FilesBase here why change it? Is it just for tests?

Reply:
Do you mean why it's
```c#
requestOptions.BaseUrl = requestOptions.BaseUrl ?? StripeConfiguration.FilesBase;
```
instead of just
```c#
requestOptions.BaseUrl = StripeConfiguration.FilesBase;
```
? If so, it's just because I think it's bad practice to silently overwrite an existing value. In practice, this doesn't change anything: `BaseUrl` is `internal`, so it cannot be set by library users, and there is no internal code that calls `FileService.Create / CreateAsync`.

Reply:
Yeah that's mostly what I meant. The dichotomy between what you know you want and what the other property could be! But all good!